jest.config.js

```diff
@@ -0,0 +1,69 @@
+const { defaults } = require('jest-config')
+module.exports = {
+    // A set of global variables that need to be available in all test environments
+    globals: {},
+
+    // This option sets the URL for the jsdom environment. It is reflected in properties such as location.href
+    testURL: 'http://localhost:8000',
+
+    // Indicates whether the coverage information should be collected while executing the test
+    collectCoverage: false,
```

Comment:
I enabled it.
this way each time we run our tests, in the end, it will run coverage so I thought maybe we don't need it, and instead, I added another command to run coverage separately. Should I delete the other command as well? (`npm run jest:coverage`)

Reply:
good job man

Reply:
This would be the better approach

"I added another command to run coverage separately. Should I delete the other command as well? (npm run jest: coverage)"

Reply:
I also think the script that you've provided to collect coverage separately, is good enough